Runtime/Scripts/Export/GameObjectExport.cs

```diff
@@ -187,7 +188,9 @@ public class GameObjectExport {
                 }
             } else
             if (gameObject.TryGetComponent(out SkinnedMeshRenderer smr)) {
-                if (smr.enabled || m_Settings.disabledComponents) {
+                if (smr.enabled || m_Settings.disabledComponents)

```

Comment:
nitpick: Again a white-space change. I know that stuff like this is pure taste, but for the sake of annotations please don't change existing formatting.